.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: tests
+
+on:
+  push:
+    branches-ignore: [main]
+  workflow_dispatch:
+
+jobs:
+  linux-unit-tests:
+    strategy:
+      matrix:
+        node: [10, 12, 14, 16]
```

Comment:
I don't see any reason to test this back to 10 and 12 is questionable https://endoflife.date/nodejs
Do we have a story to support 18 which is the current LTS?

Reply:
Very good point, I'll create a follow up story to figure out node versioning support